packages/api/endpoints/reconciliation-reports.js

```diff
@@ -79,6 +79,7 @@ async function getReport(req, res) {
     if (Key.endsWith('.json') || Key.endsWith('.csv')) {
       const reportSize = await getObjectSize({ s3: s3(), bucket: Bucket, key: Key });
       // estimated payload size, add extra
+      //  this was an intentional change per Jonathan's PR comment #2976
```

Comment:
I don't think we need this comment. It looks like Jonathan was asking if the change on line 86 was intentional in the `main` to `feature/rds` merge https://github.com/nasa/cumulus/pull/2976/files#r889022958

It looks like a whitespace change caused by a different linter configuration.

Reply:
I think line 25 in this commit (https://github.com/nasa/cumulus/commit/9be1fa7ea443838ca9c121e1f7ef40ac988a0e24#diff-a116c50394a9d17b72826a6dc87a35d7ca471c46de55657d67d5a16da2e68cb0) which has been pushed down to line 30 now (https://github.com/nasa/cumulus/blob/d044595cdb671d32b5f9b5bbf04540c8e1614686/packages/api/endpoints/reconciliation-reports.js#L30) is what he was talking about. :) 
I had to trace way back because I recall it made more sense (than just a whitespace) when he told me at the time. But I could be wrong tho. So I should still remove this comment right?

Reply:
Comment is removed.